### ISSUE TITLE
Cannot find path 

### ISSUE THREAD

**Issue by rpherman** [hexops/mach#517]

**Comment by alichraghi** [comment]
could u provide some more info?

**Comment by rpherman** [comment]
> could u provide some more info?

I just edited my issue with the proper error I received. Thanks!

**Comment by rpherman** [comment]
I am on Windows 11. I turned on Developer Mode and set git to use symlinks per the Known Issues page.

It runs a bit further and then spits all of this out:

`PS E:\> cd mach
PS E:\mach> zig build run-example-boids
.\build.zig:205:9: error: no member named 'source' in struct 'std.build.Pkg'
        .source = .{ .path = "examples/libs/zmath/src/zmath.zig" },
        ^
.\libs\freetype\build.zig:12:5: error: no member named 'source' in struct 'std.build.Pkg'
    .source = .{ .path = thisDir() ++ "/src/main.zig" },
    ^
.\libs\ecs\build.zig:14:15: error: no member named 'use_stage1' in struct 'std.build.LibExeObjStep'
    main_tests.use_stage1 = true;
              ^
.\libs\sysaudio\build.zig:15:5: error: no member named 'source' in struct 'std.build.Pkg'
    .source = .{ .path = thisDir() ++ "/soundio/main.zig" },
    ^
.\build.zig:402:5: error: no member named 'source' in struct 'std.build.Pkg'
    .source = .{ .path = thisDir() ++ "/src/main.zig" },
    ^
.\build.zig:416:10: error: type '@typeInfo(@typeInfo(@TypeOf(std.child_process.ChildProcess.init)).Fn.return_type.?).ErrorUnion.error_set!*std.child_process.ChildProcess' does not support field access
    child.cwd = (comptime thisDir());
         ^
.\libs\glfw\build.zig:157:10: error: type '@typeInfo(@typeInfo(@TypeOf(std.child_process.ChildProcess.init)).Fn.return_type.?).ErrorUnion.error_set!*std.child_process.ChildProcess' does not support field access
    child.cwd = thisDir();
         ^
.\libs\gpu-dawn\sdk.zig:157:18: error: type '@typeInfo(@typeInfo(@TypeOf(std.child_process.ChildProcess.init)).Fn.return_type.?).ErrorUnion.error_set!*std.child_process.ChildProcess' does not support field access
            child.cwd = comptime thisDir();
                 ^
.\libs\basisu\build.zig:106:10: error: type '@typeInfo(@typeInfo(@TypeOf(std.child_process.ChildProcess.init)).Fn.return_type.?).ErrorUnion.error_set!*std.child_process.ChildProcess' does not support field access
    child.cwd = (comptime thisDir());
         ^
.\libs\sysaudio\build.zig:135:10: error: type '@typeInfo(@typeInfo(@TypeOf(std.child_process.ChildProcess.init)).Fn.return_type.?).ErrorUnion.error_set!*std.child_process.ChildProcess' does not support field access
    child.cwd = (comptime thisDir());
         ^
.\libs\glfw\system_sdk.zig:246:10: error: type '@typeInfo(@typeInfo(@TypeOf(std.child_process.ChildProcess.init)).Fn.return_type.?).ErrorUnion.error_set!*std.child_process.ChildProcess' does not support field access
    child.cwd = cwd;
         ^
.\libs\gpu-dawn\sdk.zig:452:18: error: type '@typeInfo(@typeInfo(@TypeOf(std.child_process.ChildProcess.init)).Fn.return_type.?).ErrorUnion.error_set!*std.child_process.ChildProcess' does not support field access
            child.cwd = comptime thisDir();
                 ^
PS E:\mach>`

**Comment by michal-z** [comment]
You need newer version of Zig compiler.

**Comment by rpherman** [comment]
> You need newer version of Zig compiler.

Do I need to build it? The Zig page had this as the latest from master.

I will try a Choco install for Windows. Thanks!

**Comment by michal-z** [comment]
I don't think you have the latest. Latest is here: https://ziglang.org/builds/zig-windows-x86_64-0.10.0-dev.3885+a7661f115.zip

**Comment by rpherman** [comment]
> I don't think you have the latest. Latest is here: https://ziglang.org/builds/zig-windows-x86_64-0.10.0-dev.3885+a7661f115.zip

Ok, I'll try this one, but it looks very similar to what I had: zig version 0.10.0-dev.980+db82c1b98

**Comment by rpherman** [comment]
No luck. I did the first step on the "Known issues" page by turning on Developer Mode in Windows and enabling symlinks in git.
The recommended step has "cd mach/gpu". I don't have a mach/gpu directory in my cloned repository of mach. Any ideas or more steps to carry out the first recommendation? I have WSL, but I am not sure what the last step "Run the exe in zig-out/bin/ outside of WSL" means.

```
Windows does not have symlinks enabled, or Git is not configured to use them. This is very annoying and [has been reported to Microsoft](https://twitter.com/slimsag/status/1508114938933362688).

Two solutions:

    (recommended) Build a native Windows binary by cross-compiling from WSL -> Windows:

    cd mach/gpu
    zig build -Dtarget=x86_64-windows
    Run the exe in zig-out/bin/ outside of WSL.

    Enable symlinks in Windows:

    [Turn on Development Mode](https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development)
    [Ensure symlinks are installed in Git](https://stackoverflow.com/a/59761201) git config --global core.symlinks true
    Re-clone the repository and try again.
```

**Comment by michal-z** [comment]
The latest is zig version 0.10.0-dev.**3885**+db82c1b98 (not zig version 0.10.0-dev.**980**+db82c1b98). You are using very old version. Looks like you have the old version in the path.

**Comment by alichraghi** [comment]
@rpherman github issues isn't a good place to help you. please consider to join [Mach's Discord](https://discord.gg/XNG3NZgCqp) server

**Comment by rpherman** [comment]
@alichraghi Sorry, not very experienced with Github. I will look there. Thanks.

**Comment by slimsag** [comment]
Excited to see you in the discord! :) We can definitely help quicker / more interactively there, GitHub issues are a bit slower to respond to for things like this. I'll close this for now.
